examples/mnist/mnist.py

```diff
@@ -58,30 +59,61 @@ def run(train_batch_size, val_batch_size, epochs, lr, momentum, log_interval):
                                                      'nll': Loss(F.nll_loss)},
                                             device=device)
 
+    @trainer.on(Events.ITERATION_STARTED)
+    def log_training_loss_start(engine):
```

Comment:
@rave78 I think it would be better have a single log function `log_training_loss` instead of two `log_training_loss_start` and `log_training_loss_end`. What do you think ?

Reply:
I think that makes total sense to merge both functions.  There is no difference starting the tqdm pbar at the ITERATION_STARTED or ITERATION_COMPLETED.